code/parachain/frame/staking-rewards/src/test/mod.rs

```diff
@@ -1378,6 +1378,7 @@ fn claim_with_insufficient_pot_funds() {
 				owner: BOB,
 				fnft_collection_id: STAKING_FNFT_COLLECTION_ID,
 				fnft_instance_id: bob_stake_id,
+				claimed_amount: BTreeMap::from([(USDT::ID, 900000)]),
```

Comment:
would be helpful to explain where these claimed amounts come from, or use a function to get the value (i.e. claim_of_stake or something similar)

Reply:
also `900000` should be `900_000`